web/schemas/ID1A3.json

```diff
@@ -64,12 +64,20 @@
     },
     {
         "key": "StaffScientist",
-        "type": "string",
+        "type": "list_string",
```

Comment:
the `list_string` is inconsistent with `list_str` data-type, please fix it to be `list_str`

Reply:
@vkuznet This has now been fixed.